maquetado/index.html

```diff
@@ -2,32 +2,66 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
- <title>HTML & CSS Training</title>
- <meta name="description" content="training">
- <meta name="author" content="wolox">
- <link rel="stylesheet" type="text/css" href="application.css">
+  <title>books</title>
+  <link rel="stylesheet" type="text/css" href="application.css">
 </head>
 <body>
-  <div class="book">
-    <img src="images/book-cover.png" class="book-image" alt="book cover"/>
-    <div class="book-info">
-      <div class="book-title">
-        <h2 class="text-large text-bold book-element-heading">Título del libro</h2>
-        <h3 class="text-grey">(género)</h3>
-      </div>
-      <div class="book-element">
-        <h3 class="text-bold book-element-heading">Autor del libro:</h3>
-        <h3 class="text-grey">Nombre del autor del libro</h3>
-      </div>
-      <div class="book-element">
-        <h3 class="text-bold book-element-heading">Editorial:</h3>
-        <h3 class="text-grey">Nombre de la editorial</h3>
-      </div>
-      <div class="book-element">
-        <h3 class="text-bold book-element-heading">Año de publicación:</h3>
-        <h3 class="text-grey">Año de publicación</h3>
-      </div>
-    </div>
-  </div>
+  <ul class="book-list">
+    <li class="book-thumbnail">
+      <a href="book.html">
+        <img src="images/book-cover.png" class="book-thumbnail-image">
+        <p class="text-large text-bold book-thumbnail-element">Título del libro</p>
+        <p class="text-grey book-thumbnail-element">Autor del libro</p>
+      </a>
+    </li>
```

Comment:
We use `li` when we are talking about a list of elements. Like a list of links or names, etc. Here we have a list of components so we use a normal `div` tag. Also, remember that the title should use `h2` instead.

Reply:
@jtorreguitar